packages/launchdarkly-js-sdk-common/src/messages.js

```diff
@@ -12,7 +12,7 @@ export const clientNotReady = function() {
 };
 
 export const eventWithoutUser = function() {
-  return 'Be sure to call `identify` in the LaunchDarkly client: http://docs.launchdarkly.com/docs/running-an-ab-test#include-the-client-side-snippet';
+  return 'Be sure to call `identify` in the LaunchDarkly client: https://docs.launchdarkly.com/docs/js-sdk-reference#section-users';
```

Comment:
I realize that the link needs to be changed, but the section that the new link points to doesn't seem to say anything about `identify`. It's just about the general concept of users.